cosmosdb/cosmosdb_data_mappings.bal

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+function convertToDatabaseResponse(json jsonResponse) returns DatabaseResponse {
+    DatabaseResponse databaseResponse = {};
+    databaseResponse.database = convertToDatabase(jsonResponse);
```

Comment:
Can't we merge lines 18 and 19?
```ballerina
    DatabaseResponse databaseResponse = { database: convertToDatabase(jsonResponse)  };
```

Reply:
Applicable to other similar record initializations too. We seem to be creating empty records and then setting the values where it can be done in one line.

Shall we update them if there was no particular reason to do so? We can use appropriate indentation to improve clarity if necessary.